cortex/experiment/model.py

```diff
@@ -160,31 +163,41 @@ def to_json(self):
         """
         A json representation of the run.
         """
-        doc = {'id': self.id, 'startTime': self.start_time, 'endTime': self.end_time, 'took': self.took,
-               'params': self.params, 'metrics': self.metrics, 'meta': self.meta,
-               'artifacts': [name for name in self.artifacts.keys()]}
+        doc = {
+            "id": self.id,
+            "startTime": self.start_time,
+            "endTime": self.end_time,
+            "took": self.took,
+            "params": self.params,
+            "metrics": self.metrics,
+            "meta": self.meta,
+            "artifacts": list(self.artifacts),
+        }
         return doc
 
     @staticmethod
-    def from_json(json, experiment):
+    def from_json(json: dict, experiment: str):
```

Comment:
experiment_id ?

Reply:
We are using `experiment_name` everywhere as the identifier for experiments. Can make this change here as well